spec/user_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative './rails_helper'
```

Comment:
**You did great job! I can see you put a great efford into this milestone.** 

- [ ] However, in your tests there are 9 failing tests in it. Due to this reason, I was not able to review your code accordingly. Please fix this issues and resubmit your code. Also, I strongly suggest you to run `rails db:reset` before you start fixing up your code. Because I also noticed some issues in your migration, and it would be great if you could look though it as well :)


<img width="1182" alt="Screenshot 2023-05-19 at 01 16 05" src="https://github.com/LeslieAine/blog-app1/assets/69011963/b90c95ef-66b3-409e-b51e-5a370e2178b1">